shared/src/commonMain/kotlin/co/nimblehq/avishek/phong/kmmic/domain/model/Answer.kt

```diff
@@ -0,0 +1,22 @@
+package co.nimblehq.avishek.phong.kmmic.domain.model
+
+interface Answerable {
+
```

Comment:
Don't need to have a blank line here

Reply:
solved in [b24d7ca](https://github.com/nimblehq/avishek-phong-kmm-ic/pull/119/commits/b24d7ca82e18d29a948702230f6b3798cbd7ab08)